.github/workflows/build-binaries.yml

```diff
@@ -118,7 +116,7 @@ jobs:
   # We want a mac arm64 build, and according to this https://github.com/actions/runner-images#available-images macos-14 is always arm64
   # macos-14 is disabled for now as we hit our free tier limit for macos builds
   build_macos_x64:
-    runs-on: macos-12
+    runs-on: macos-13
```

Comment:
Sanity Check: If we update this will the app still run on macOS 12? It should be fine unless we update electron again right?

Reply:
no, it will probably not work on macos12 anymore.

Reply:
but gh removed macos-12, so we have to too.

Reply:
Okay then we will also need to update our minimum requirements and let marketing know before our next release. @Bilb